crates/nu_plugin_polars/src/dataframe/series/string/str_lengths.rs

```diff
@@ -56,17 +89,42 @@ impl PluginCommand for StrLengths {
         call: &EvaluatedCall,
         input: PipelineData,
     ) -> Result<PipelineData, LabeledError> {
-        command(plugin, engine, call, input).map_err(LabeledError::from)
+        let value = input.into_value(call.head)?;
+        match PolarsPluginObject::try_from_value(plugin, &value)? {
+            PolarsPluginObject::NuDataFrame(df) => command_df(plugin, engine, call, df),
+            PolarsPluginObject::NuLazyFrame(lazy) => {
+                command_df(plugin, engine, call, lazy.collect(call.head)?)
+            }
+            PolarsPluginObject::NuExpression(expr) => command_expr(plugin, engine, call, expr),
+            _ => Err(cant_convert_err(
+                &value,
+                &[PolarsPluginType::NuDataFrame, PolarsPluginType::NuLazyFrame],
+            )),
+        }
+        .map_err(LabeledError::from)
     }
 }
 
-fn command(
+fn command_expr(
+    plugin: &PolarsPlugin,
+    engine: &EngineInterface,
+    call: &EvaluatedCall,
+    expr: NuExpression,
+) -> Result<PipelineData, ShellError> {
+    let res: NuExpression = if call.has_flag("bytes")? {
+        expr.into_polars().str().len_bytes().into()
+    } else {
+        expr.into_polars().str().len_chars().into()
+    };
+    res.to_pipeline_data(plugin, engine, call.head)
+}
+
+fn command_df(
```

Comment:
Does this code path/function need to account for if `call.has_flag("bytes")`?

Reply:
probably a good idea, since it looks like it is supported on series too. I'll add it.